accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloRDDWriter.scala

```diff
@@ -33,8 +33,8 @@ object AccumuloRDDWriter {
         // Map and sort first, so that partitioner is carried over to groupByKey
         raster
           .map { case (key, value) => (encodeKey(key), (key, value)) }
-          .sortByKey()
           .groupByKey()
+          .sortByKey()
```

Comment:
Using `.sortByKey().groupByKey()` the order is not guaranteed actually, only which keys end up in each partition; reducers may fetch tasks in an arbitrary order, depending on which ones are available first, that's why to guarantee ordering we should sort after group.